src/thor/dijkstras.cc

```diff
@@ -364,10 +364,16 @@ void Dijkstras::Compute(google::protobuf::RepeatedPtrField<valhalla::Location>&
     // Check if we should stop
     cb_decision = ShouldExpand(graphreader, pred, expansion_direction);
     if (cb_decision != ExpansionRecommendation::prune_expansion) {
-      // Expand from the end node in forward direction.
+      // Expand from the end node in expansion_direction.
       ExpandInner<expansion_direction>(graphreader, pred.endnode(), pred, predindex, opp_pred_edge,
                                        false, time_infos.front());
     }
+
+    if (expansion_callback_) {
+      // "algorithm" arg defaults already to dijkstra
+      expansion_callback_(graphreader, pred.edgeid(), "unidirectional_dijkstra", "s",
```

Comment:
can you change this to just `dijkstras` i think a test above checks for this as well

Reply:
changed name to dijkstras :+1: